app/Http/Controllers/CommentController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Book;
+use App\User;
+use App\Comment;
+
+
+class CommentController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $books = Book::all();
+        return view('comments.index', ['books' => $books]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create(Request $request)
+    {
+        $books = Book::all();
+        $users = User::all();
+
+        $data = [
+            'book_id' => $request->book_id,
+            'users' => $users,
+            'books' => $books
+        ];
+        return view('comments/create', $data);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $request->validate(['message' => 'required']);
+
+        // $comment = new Comment;
+
+        Comment::create(
+            [
+                'message' => $request->message,
+                'book_id' => $request->book_id,
+                'user_id' => $request->user_id
```

Comment:
```suggestion
                'user_id' => $request->user_id,
```
配列の最後の要素にはカンマを付けるようにしましょう。
コピペする時や、配列を書き換える時に楽なので

Reply:
レビューありがとうございます。
修正いたしました。